WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -392,6 +404,18 @@ finishedCoinJoin.CoinJoinTask.Status switch
 					&& x.Kitchen.HasIngredients)
 			.ToImmutableDictionary(x => x.WalletName, x => x);
 
+	private bool IsWalletPrivate(Wallet wallet)
+	{
+		var coins = new CoinsView(wallet.Coins);
+
+		if (GetPrivacyPercentage(coins, wallet.KeyManager.AnonScoreTarget) > 0.99)
```

Comment:
Since https://github.com/zkSNACKs/WalletWasabi/pull/8025 no need for the hack anymore.

```suggestion
		if (GetPrivacyPercentage(coins, wallet.KeyManager.AnonScoreTarget) == 1)
```